spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,161 @@
+      column_coordinate: 4,
+      color: "white"
+      )
+
+    # Build black pieces
+    (0..7).each do |i|
+      Pawn.create(
+        row_coordinate: 1,
+        column_coordinate: i,
+        color: "black"
+        )
+    end
+
+      rook1 = Rook.create(
+        row_coordinate: 0,
+        column_coordinate: 0,
+        color: "black"
+        )
+
+      rook2 = Rook.create(
+        row_coordinate: 0,
+        column_coordinate: 7,
+        color: "black"
+        )
+
+      knight1 = Knight.create(
+        row_coordinate: 0,
+        column_coordinate: 1,
+        color: "black"
+        )
+
+      knight2 = Knight.create(
+        row_coordinate: 0,
+        column_coordinate: 6,
+        color: "black"
+        )
+
+      bishop1 = Bishop.create(
+        row_coordinate: 0,
+        column_coordinate: 2,
+        color: "black"
+        )
+
+      bishop2 = Bishop.create(
+        row_coordinate: 0,
+        column_coordinate: 5,
+        color: "black"
+        )
+
+      queen = Queen.create(
+        row_coordinate: 0,
+        column_coordinate: 3,
+        color: "black"
+        )
+
+      king = King.create(
+        row_coordinate: 0,
+        column_coordinate: 4,
+        color: "black"
+        )
+
+      expect(king.row_coordinate).to eq(0)
+      expect(king.column_coordinate).to eq(4)
+      expect(queen.row_coordinate).to eq(0)
+      expect(queen.column_coordinate).to eq(3)
+      expect(bishop1.row_coordinate).to eq(0)
+      expect(bishop1.column_coordinate).to eq(2)
+      expect(bishop2.row_coordinate).to eq(0)
+      expect(bishop2.column_coordinate).to eq(5)
+      expect(rook1.row_coordinate).to eq(0)
+      expect(rook1.column_coordinate).to eq(0)
+      expect(rook2.row_coordinate).to eq(0)
+      expect(rook2.column_coordinate).to eq(7)
+      expect(knight1.row_coordinate).to eq(0)
+      expect(knight1.column_coordinate).to eq(1)
+      expect(knight2.row_coordinate).to eq(0)
+      expect(knight2.column_coordinate).to eq(6)
+      expect(game.pawns.row_coordinate).to eq(0)
+
+      expect(white_king.row_coordinate).to eq(7)
+      expect(white_king.column_coordinate).to eq(4)
+      expect(white_queen.row_coordinate).to eq(7)
+      expect(white_queen.column_coordinate).to eq(3)
+      expect(white_bishop1.row_coordinate).to eq(7)
+      expect(white_bishop1.column_coordinate).to eq(2)
+      expect(white_bishop2.row_coordinate).to eq(7)
+      expect(white_bishop2.column_coordinate).to eq(5)
+      expect(white_rook1.row_coordinate).to eq(7)
+      expect(white_rook1.column_coordinate).to eq(0)
+      expect(white_rook2.row_coordinate).to eq(7)
+      expect(white_rook2.column_coordinate).to eq(7)
+      expect(white_knight1.row_coordinate).to eq(7)
+      expect(white_knight1.column_coordinate).to eq(1)
+      expect(white_knight2.row_coordinate).to eq(7)
+      expect(white_knight2.column_coordinate).to eq(6)
+      expect(game.pawns.row_coordinate).to eq(7)
+    end
+  end
+end
```

Comment:
The test for `populate_board!` should go into `spec/models/game_spec.rb`

And you can structure it like this.

```
describe '#populate_board!' do
  it 'places correct number of pawns on the board' do
    game = FactoryGirl.create(:game) # you'll need to setup FactoryGirl

    game.populate_board!

    expect(game.pawns.white.count).to eq 8
    expect(game.pawns.black.count).to eq 8
  end
end
```

^ This is just one example test. You can do this for all the types of pieces and get more detailed by checking their positions.